### ISSUE TITLE
Cut/Copy/Paste not working correctly with Progression Levels

### ISSUE THREAD

**Issue by gloverkari** [cassproject/cass-editor#1261]
The UI shows "Cut Competencies" . When "Cut Competencies" is selected with a checkmark adjacent to a level, it doesn't delete the level. Just as with competencies, the more you check and uncheck competencies to be able to add rather than edit, the more it goes haywire. I select one level an it shows edit multiple instead of add. All of these issues are in screenshots below.

The copy/cut/paste should not show copy/cut/paste competencies.

After adding just a couple of sub levels, the UI went haywire and would not let me add anything. I recorded this in a video. It's the same issues Stuart recorded awhile ago showing how as you work on anything after a short period of time, things begin to go haywire to the point that you cannot edit, add, move, etc..

![211125695-818b8b02-9455-43e3-8453-f6b7430dc553](https://user-images.githubusercontent.com/78623855/211604269-3e552920-0df2-4c51-a970-38a815ff3d09.png)

https://sandbox.credentialengine.org/cass-editor/?user=wait&link=true&newObjectEndpoint=https://sandbox.credentialengineregistry.org/resources/ce-&server=https://dev.cassproject.org/api/&ceasnDataFields=true&show=all&concepts=true&origin=https://sandbox.credentialengine.org&action=import#/progressionLevels?user=wait&link=true&newObjectEndpoint=https%3A%2F%2Fsandbox.credentialengineregistry.org%2Fresources%2Fce-&server=https%3A%2F%2Fdev.cassproject.org%2Fapi%2F&ceasnDataFields=true&show=all&concepts=true&origin=https%3A%2F%2Fsandbox.credentialengine.org&action=import?concepts=true&frameworkId=https://sandbox.credentialengineregistry.org/resources/ce-27268c79-de7b-4ea6-a9cd-210b0a6eda6b

![211125787-97c3f4e2-8342-4459-a0d4-3ea28498752b (1)](https://user-images.githubusercontent.com/78623855/211606258-177f152b-4f67-48f2-8438-f5348452f474.png)

![211125924-39e4cc91-8b6a-4caa-b6ff-224010468d70](https://user-images.githubusercontent.com/78623855/211606460-173413c4-de47-4435-a06d-dfca596a8dea.png)

![211126133-b473aeae-8d99-4813-bffc-b5851c663c98](https://user-images.githubusercontent.com/78623855/211605723-368e2f4e-7ce1-4f0c-990d-c180d85a8b35.png)

**Comment by gloverkari** [comment]
At the heart of this issue is the current behavior of copy/paste functionality across all objects. Paste, by default, creates a link to the original object being copied.  This functionality makes sense for competencies, but not for progression levels. A progression cannot have multiple links to the same progression level because each level self-defines its location in the progression. 

For now, I've removed the Copy button functionality from progressions (to avoid unwanted behavior) and am awaiting decisions for how to rework the functionality. The new functionality will need to be consistent across all objects.

We now have an internal issue and discussion around this need and I have opened a new issue in github to mirror it: https://github.com/cassproject/cass-editor/issues/1297

Feedback and suggestions from the Credential Engine team would be very much appreciated.
